pkg/apisix/plugin_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"testing"
 
 	"github.com/apache/apisix-ingress-controller/pkg/metrics"
+
```

Comment:
Can we sort these imports?

Reply:
My local golangci-lint would print these warnings below:
```sh
golangci-lint run
pkg/apisix/plugin_test.go:25: File is not `goimports`-ed with -local github.com/apache/apisix-ingress-controller (goimports)
	"github.com/apache/apisix-ingress-controller/pkg/metrics"
pkg/apisix/schema_test.go:24: File is not `goimports`-ed with -local github.com/apache/apisix-ingress-controller (goimports)
	"github.com/apache/apisix-ingress-controller/pkg/metrics"
```
Finally, I found that if I split the imports then there no warnings left.
Maybe I should ignore it in the next commits.

Reply:
I think it's caused by the lexicographical orders.